lang/en/docs/properties-directory/non-scalar/file-content.md

```diff
@@ -0,0 +1,17 @@
+# File Content
+
+<span class="btn badge b-success border-50">Non-Scalar</span> <span class="btn badge b-info border-50">Electronic</span>
```

Comment:
File content is not an electronic property

Reply:
Fixed